src/getHumanAge.test.js

```diff
@@ -7,4 +7,39 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array with two elements', () => {
+    expect(getHumanAge(1, 1))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 0 if both
+    catAge and dogAge are less than 15`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 1 if both
+    catAge and doqAge are between 15 and 23`, () => {
```

Comment:
Typo found in 'doqAge', it should be 'dogAge'. Make sure to keep the variable names correct and consistent. Fix all cases please